options/locale/locale_en-US.ini

```diff
@@ -1767,6 +1767,8 @@ pulls.auto_merge_canceled_schedule_comment = `canceled auto merging this pull re
 pulls.delete.title = Delete this pull request?
 pulls.delete.text = Do you really want to delete this pull request? (This will permanently remove all content. Consider closing it instead, if you intend to keep it archived)
 
+pulls.recently_pushed_new_branches = You pushed to <strong>%[1]s/%[2]s</strong> on branch <strong>%[3]s</strong> %[4]s
```

Comment:
Isn't the repository a little redundant to mention if the hint only ever shows up on the repository home page?

Reply:
Fixed